CONTRIBUTING.md

```diff
@@ -9,16 +9,21 @@ The current areas of focus are:
 
 ## Table of Contents 📖
 
-1. [Roadmap](#roadmap)
-1. [Support questions & reporting bugs](#support-questions--reporting-bugs)
-1. [Development](#development)
-    1. [Project structure](#project-structure)
-    1. [Environment setup](#environment-setup)
-    1. [Coding style](#coding-style)
-    1. [Checks & Testing](#checks--testing)
-    1. [Docs](#docs)
-1. [Release](#release)
-1. [Finally](#finally)
+
```

Comment:
I prefer it the way it was. More concise. No one is going to look for the table of contents while reading the table of contents.